objects/src/accounts/data.rs

```diff
@@ -0,0 +1,212 @@
+use std::{
+    fs::{self, File},
+    io::{self, Read},
+    path::Path,
+};
```

Comment:
The errors above is because this crate has no-std support. On the build without std, namely wasm, it fails.

If you need the path manipulation, then my suggestion is to move this behind a feature flag, [e.g. `#[cfg(feature = "std")]`](https://github.com/0xPolygonMiden/miden-base/blob/bb00c3038035d70d3f1e96c161b251ec29ad1434/objects/src/errors.rs#L250-L251)

Reply:
Added.